src/toil/resource.py

```diff
@@ -378,7 +378,8 @@ def forModule(cls, name):
                 dirPathTail = filePath.pop()
                 assert dirPathTail == package
             dirPath = os.path.sep.join(filePath)
-        assert os.path.isdir(dirPath)
+        require(os.path.isdir(dirPath),
+                'Bad directory path %s for module %s. Maybe it contains egg-links?', dirPath, name)
```

Comment:
In 

> Maybe it contains egg-links?

it's unclear what "it" refers to. It can't be the value of `dirPath` because we've just established that that's not a directory, so it can't contain things.

To make it clear, I'm asking you to replace `Maybe it contains egg-links? ` with `Note that hot-deployment does not support .egg-link files yet.`